Unosquare.CodeChallenge/WarehouseModels/Models/Company.cs

```diff
@@ -12,7 +12,7 @@ public class Company : ICompany, IModel
         private int _id;
         private string _name;
 
-        public int Id { get => _id; set => _id=value; }
-        public string Name { get => _name; set => _name=value; }
+        public int id { get => _id; set => _id=value; }
```

Comment:
Oh I see,  my apologies, I meant to say the standard in the front is to have the first letter lowercase, but in the backend Properties should be PascalCase